.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
```

Comment:
Probably could be 100

Reply:
This is the default I copied from other repos. :man_shrugging: